### PR TITLE
added static content if no returns

### PR DESCRIPTION
**Pull request by willdot**
Task: 70889

### CODE REVIEW

File: src/EA.Weee.Web/Areas/AatfReturn/Views/Returns/Index.cshtml [hunk 1]
```diff
@@ -57,84 +61,91 @@
...
-
-            @@media only screen and (max-width: 760px), (min-device-width: 768px) and (max-device-width: 1024px) {
-                td:nth-of-type(1):before {
-                    content: "Year";
-                }
+    }
+    <p>
+        @Html.ActionLink("Perform another activity", "ChooseActivity", "Home", new { area = "Scheme", pcsId = @Model.OrganisationId }, null)
+    </p>
+    <style type="text/css">
```
Comment (JB12298) [inline src/EA.Weee.Web/Areas/AatfReturn/Views/Returns/Index.cshtml]: Can this css be moved to the SASS files?

Comment (willdot) [reply src/EA.Weee.Web/Areas/AatfReturn/Views/Returns/Index.cshtml]: @JohnPadgettCivica  How do we go about moving this to the SASS file? Not sure what this css is doing.....

Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/AatfReturn/Views/Returns/Index.cshtml]: We dont need to move this to SASS as its specific to this page and table

Comment (willdot) [reply src/EA.Weee.Web/Areas/AatfReturn/Views/Returns/Index.cshtml]: Awesome!

**Pull request merged**